daml-script/test/BUILD.bazel

```diff
@@ -157,6 +159,10 @@ da_scala_test_suite(
         ":script-test-1.dev.dar",
         ":script-test-no-ledger.dar",
         "//ledger/test-common/test-certificates",
+        "@openssl_dev_env//:bin/openssl" if not is_windows else "@openssl_dev_env//:usr/bin/openssl.exe",
+    ],
+    jvm_flags = [
+        """-Djava.security.debug=\"certpath ocsp\"""",  # This facilitates debugging of the OCSP checks mechanism
```

Comment:
I don't think you need the triple-quotes here.

Can you use single quotes to avoid escaping, or does that not fly in Skylark?